features/dd-sdk-android-logs/src/main/kotlin/com/datadog/android/log/Logs.kt

```diff
@@ -34,4 +35,20 @@ object Logs {
 
         sdkCore.registerFeature(logsFeature)
     }
+
+    /**
+     * Identify whether a [Logs] has been enabled for the given SDK instance.
+     *
+     * This check is useful in scenarios where more than one component may be responsible
+     * for enabling the feature
+     *
+     * @param sdkCore the [SdkCore] instance to check against. If not provided, default instance
+     * will be checked.
+     * @return whether Logs has been enabled
+     */
+    @JvmOverloads
+    @JvmStatic
+    fun isEnabled(sdkCore: SdkCore = Datadog.getInstance()): Boolean {
```

Comment:
Seems like implementation is different from iOS, where this method is public, but is still in the internal extension? Here we expose it directly, without any proxies.

Reply:
Yes, I can make it internal but I based the decision based on the documentation on `GlobalRumMonitor`. I figured Android might have more of a need for such a method than iOS, but if you think not so much I can make the internal proxy.

Reply:
I'm ok with having it as public, just was curious about the consistency of public API with iOS.